content/docs/start/experiments-trail/experiments.md

```diff
@@ -0,0 +1,319 @@
+`train.py`.
+
+```dvc
+$ dvc exp run --set-param model.conv_units=24
+...
+Reproduced experiment(s): exp-6c06d
+Experiment results have been applied to your workspace.
+
+To promote an experiment to a Git branch run:
+
+        dvc exp branch <exp>
+```
+
+When you run `dvc exp run` with `--set-param`, it updates the parameter file. We
+can see the effect of it by looking at the diff.
+
+```dvc
+$ git diff params.yaml
+```
+
+```git
+-model:
+-  conv_units: 16
++model:
++  conv_units: 24
+```
+
+## 🏃‍♂️🏃🏾‍♂️🏃🏻‍♂️ Run multiple experiments in parallel
+
+Instead of running the experiments one-by-one, we can define them to run in a
+batch. This is especially handy when you have long running experiments.
+
+We add experiments to the queue using the `--queue` option of `dvc exp run`. We
+also use `-S` (`--set-param`) to set a value for the parameter.
+
+```dvc
+$ dvc exp run --queue -S model.conv_units=32
+Queued experiment '6518f17' for future execution.
+$ dvc exp run --queue -S model.conv_units=64
+Queued experiment '30eb9b2' for future execution.
+$ dvc exp run --queue -S model.conv_units=128
+Queued experiment 'ac66940' for future execution.
+$ dvc exp run --queue -S model.conv_units=256
+Queued experiment '8bb6049' for future execution.
+```
+
+Next, run all (`--run-all`) queued experiments in parallel. You can specify the
+number of parallel processes using `--jobs`:
+
+```dvc
+$ dvc exp run --run-all --jobs 2
+```
+
+## ↔️ Comparing experiments
+
+The experiments are run several times with different parameters. We use
+`dvc exp show` to compare all of these experiments. This command presents the
+parameters and metrics produced in experiments in a nicely formatted table.
+
+```dvc
+$ dvc exp show
+```
+
+![](/img/start-dvc-exp-show-210704.png)
+
+By default it shows all the parameters and the metrics with the timestamp. If
+you have large number of parameters, metrics or experiments, this may lead to a
+cluttered view. You can limit the table to specific metrics, or parameters, or
+hide the timestamp column with `--include-metrics`, `--include-params`, or
+`--no-timestamp` options of the command, respectively.
+
+```dvc
+$ dvc exp show --no-timestamp \
+  --include-params model.conv_units --include-metrics acc
+```
+
+![](/img/start-dvc-exp-show-no-timestamp-210704.png)
+
+## 🔏 Persisting experiments
+
+After selecting an experiment from the table, you can commit the hyperparameters
+and other dependencies that produced this successful experiment to your Git
+history.
+
+`dvc exp apply` brings back all specific artifacts and parameters from the
+experiment to the <abbr>workspace</abbr>.
+
+```dvc
+$ dvc exp apply exp-cb13f
+Changes for experiment 'exp-cb13f' have been applied to your current workspace.
+```
+
+We can see the changes in the repository and commit them to Git.
+
+```dvc
+$ git diff
+$ git add .
+$ git commit -m "Successful experiment"
+```
+
+It may also be desirable to commit a particular experiment to a Git branch
```

Comment:
Do we need to introduce both `apply` and `branch` here? It seems sufficient to pick one as a default and save the other for the UG.

Reply:
removed `dvc exp apply` and kept `branch`, as it's the option advertised in the command output as well.